plugins/inputs/webhooks/plex/plex_webhooks.go

```diff
@@ -0,0 +1,43 @@
+package plex
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+)
+
+type PlexWebhook struct {
+	Path string
+	acc  telegraf.Accumulator
+}
+
+func (p *PlexWebhook) Register(router *mux.Router, acc telegraf.Accumulator) {
+	router.HandleFunc(p.Path, p.eventHandler).Methods("POST")
+	log.Printf("I! Started the webhooks_plex on %s\n", p.Path)
```

Comment:
If you add `Log telegraf.Logger` to the struct: https://github.com/influxdata/telegraf/blob/817c9a69a94060aece50889c27e7c821a36a6530/plugins/inputs/influxdb_listener/http_listener.go#L77

This can become `p.Log.Infof("...) to make use of the new plugin alias functionality.
https://github.com/influxdata/telegraf/blob/817c9a69a94060aece50889c27e7c821a36a6530/plugins/inputs/influxdb_listener/http_listener.go#L206